checkstyle.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+
+<module name="Checker">
+  <module name="Header">
+    <property name="header" value="/**\n * Pokemon Go Bot  Copyright (C) 2016  PokemonGoBot-authors (see authors.md for more information)\n * This program comes with ABSOLUTELY NO WARRANTY;\n * This is free software, and you are welcome to redistribute it under certain conditions.\n *\n * For more information, refer to the LICENSE file in this repositories root directory\n */"/>
+  </module>
+
+  <module name="NewlineAtEndOfFile">
+    <property name="lineSeparator" value="lf" />
+  </module>
+
+  <module name="FileTabCharacter"/>
+  <module name="FileLength"/>
+
+
+  <!-- Trailing spaces -->
+  <module name="RegexpSingleline">
+    <property name="format" value="\s+$"/>
+    <property name="message" value="Line has trailing spaces."/>
+  </module>
+
+  <!-- Space after 'for' and 'if' -->
+  <module name="RegexpSingleline">
+    <property name="format" value="^\s*(for|if)[^ ]"/>
+    <property name="message" value="Space needed before opening parenthesis."/>
+  </module>
+
+  <!-- For each spacing -->
+  <module name="RegexpSingleline">
+    <property name="format" value="^\s*for \(.*?([^ ]:|:[^ ])"/>
+    <property name="message" value="Space needed around ':' character."/>
+  </module>
+
+  <module name="TreeWalker">
```

Comment:
For some reason not a single TreeWalker submodule check works but there's no error whatsoever.

And my googlefu is not helping me :cry: